packages/ses/src/compartment-shim.js

```diff
@@ -292,10 +293,12 @@ export const makeCompartmentConstructor = (
       intrinsics,
       newGlobalPropertyNames: sharedGlobalPropertyNames,
       makeCompartmentConstructor: targetMakeCompartmentConstructor,
-      safeEvaluate,
       markVirtualizedNativeFunction,
     });
 
+    // TODO: maybe add evalTaming to the Compartment constructor 3rd options?
```

Comment:
Seems like a good idea to me for compartments to transitively respect the `noEval` option, even if their behavior doesn’t vary between `safeEval` and `unsafeEval`. In fact, it might be seen as a security hole if something relies on our `noEval` instead of a CSP.

Reply:
I don't know how to pass the option from `lockdown` to `Compartment`. It looks like the Compartment constructor has been installed to the global before lockdown and they're two separate features.

If the Compartment creator wants `noEval`, they can replace the implementation on the `compartment.globalThis`, or as the comment said we can add a new option.